spec/unit/facter/syslog_ng_version_spec.rb

```diff
@@ -37,20 +37,26 @@
         'Enable-Linux-Caps' => 'off',
       }
       expected_syslog_ng_version = '3.7.1'
+
```

Comment:
Could you please explain the change ?
The new form doesn't make sense to me : as I understand it, the `before` fixes that `which syslog-ng` returns `/usr/bin/syslog-ng`, and the test tests what we just fixed :question:

Reply:
Both code are equivalent, but RSpec is a bit more explicit and split the 2 things that mocha do in 2 operations:
1. `allow` describe what *can* happen and stubs it;
2. `expect` ensure it happened, raising an exception if it did not.

Mocha with a single `expects` before the code being trained setup the stubs and register code to check it was called at the end of the test.

Reply:
:pray: